experiment-rollout/data.json

```
@@ -189,12 +177,33 @@
     "lockdown_mode": {
         "experimentType": 0,
         "rolloutType": 0,
-        "rate": 0
+        "rate": 100,
+        "requirements": [
+            {
+                "type": 1,
+                "value": [
+                    "MEMBER_SAFETY_PAGE_ROLLOUT"
+                ],
+                "rate": 100,
+                "ranges": [
+                    [
+                        0,
+                        10000
+                    ]
+                ]
+            }
+        ]
```

```suggestion
        "rate": 0
```
it's lockdown mode, not member safety, and seems this data is old

---

That's [what the API reports](https://rollouts.advaith.io/#2023-05_guild_member_safety_experiment), but it seems my servers that *should* have it doesn't, so I'll remove that.

---

> it's lockdown mode, not member safety, and seems this data is old

That's expressing the it's rolled out with the feature, I set the rate wrong.

---

oh sorry i saw now

---

but i don't think it's usable right now, so this is bettor for now